ffi/libwasm2oci.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"C"
+
+	"github.com/engineerd/wasm-to-oci/pkg/oci"
+)
+
+//export Pull
+func Pull(ref, outFile string) int64 {
+	err := oci.Pull(ref, outFile)
+	if err != nil {
+		return 1
```

Comment:
If this returned a Go `error`, the constructed return type in the C header would be a `GoInterface` which only contains two `void*` fields. This is rather unhelpful, so for now, the easiest way is to return an `int64` and treat it as exit code.

However, we should at least log the error message so callers have a little bit more context.